src/cmd/cro3.bash

```diff
@@ -147,6 +151,10 @@ _cro3_get_options() { # current
             _cro3_comp_fs -f "${1}"
           fi
           ;;
+        key)
+          if [ "${COMP_WORDS[1]}" == "config" ]; then
```

Comment:
Maybe we should check the COMP_WORDS[0] to be `cro3` at the same time to avoid other "config" commands to trigger this completion

Reply:
Would you consider `cro3 BRAH config` case? Here `COMP_WORDS[]` array contains the command line, so `COMP_WORDS[0]` is always the `cro3`. Or, it should be checked in `_cro3()` in general (and that is another issue).

Reply:
Ah, never mind... Yeah I thought about cro3 brah ... config case but it does not happen as you said... Please forget about this. Thank you!